Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -59,6 +59,8 @@ ZipScript.prototype.go = function () {
         .to(wine.prefixDirectory() + "/drive_c/" + this._name)
         .extract();
 
+    wine.runInsidePrefix(wine.prefixDirectory() + "/drive_c/" + this._name + ".exe");
```

Comment:
I think this will break most of our existing scripts that make use of `ZipScript`. The reason is that they don't contain an executable in the zip file that should be executed during the installation.

Take [Druid Soccer](https://github.com/PhoenicisOrg/scripts/blob/920c415c38cfffca58d3813108893d1c50c69f44/Applications/Games/Druid%20Soccer/Online/script.js) for example. This installer script only sets an executable that represents the result of the installation but is not the installer. 

In addition I don't think that `this._name` is the name of an `*.exe` file. It normally contains the name of the application. I think we need to add a new field to `ZipScript` that contains the name of the executable in the archive. I think this was also suggested by @plata in https://github.com/PhoenicisOrg/scripts/pull/908#issuecomment-508926377.

Reply:
@Zemogiter Could you test by adding 
```js
	setupPathInsidePrefix(setupPath) {
		this._setupPath = setupPath;
		return this;
	}
```
under url method and
```js
		if (this._setupPath !== 'undefined') {
			wine.runInsidePrefix(this._setupPath);
		}
``` 
inside go method after Extractor ? 
@madoar I think this should do the job.

Reply:
Yes, additionally I would recommend adding a `fileExists` check for `this._setupPath` to check that the file really exists before executing it